Formula/swiftgen.rb

```diff
@@ -13,7 +13,7 @@ class Swiftgen < Formula
   depends_on :xcode => ["8.0", :build]
 
   def install
-    rake "install[#{bin},#{lib},#{pkgshare}/templates]"
+    system("NO_CODE_LINT=1 rake 'cli:install[#{bin},#{lib},#{pkgshare}/templates]'")
```

Comment:
We need to avoid running swiftint when running the CI or when Homebrew builds the bottle (well when we don't run on a local machine) because we have no guarantee that the remote machine has the same version of swiftint that the one we have on our local machines. 

So if we let Homebrew run the swiftint phase during compilation there's a high risk of compilation failure just due to this linting phase if the swiftint versions don't match or if swiftint isn't installed. We want to avoid such failures especially as we already ran our linter locally or on our CI. 

To disable that linting phase we use an environment variable which, when set, skips the call to swiftint and avoids the risk of build failure. That's why we pass that NO_CODE_LINT environment variable there. 

If there's any way to still use `rake` instead of `system` but also set the environment variable to 1 during that rake execution I'll be happy to change to that solution. Maybe just setting `ENV['NO_CODE_LINT']=1` before there call?!